proto/storage/policy.proto

```diff
@@ -20,12 +20,12 @@ message Policy {
     string                      remediation             = 5;
     bool                        disabled                = 6 [(gogoproto.moretags) = "search:\"Disabled\""];
     repeated string             categories              = 7 [(gogoproto.moretags) = "search:\"Category,store\""];
-    PolicyFields                fields                  = 8 [(gogoproto.moretags) = 'sql:"-"'];
+    reserved                                              8; // was PolicyFields fields
```

Comment:
I don't see a reason why we should keep `PolicyFields` protobuf. Let's remove the it altogether.

Reply:
I am not sure why you pointed to https://github.com/stackrox/stackrox/pull/879#issuecomment-1063525408.

Thus far, this PR has only removed the usage of the `PolicyFields` in the `Policy` protobuf. Additionally, we should remove the definition of `PolicyFields` aka `PolicyFields` protobuf.